content/gateway_guide.md

```diff
@@ -506,46 +524,37 @@ You may also enable `RequireAuth` for your cold wallet in order to use the [Auth
 
 If ACME decides to use Authorized Accounts, ACME creates an interface for users to get their Ripple trust lines authorized by ACME's cold account. After Alice has extended a trust line to ACME from her Ripple account, she goes through the interface on ACME's website to require ACME authorize her trust line. ACME confirms that it has validated Alice's identity information, and then sends a TrustSet transaction to authorize Alice's trust line.
 
-The following is an example of using the [Ripple-REST Grant Trustline method](ripple-rest.html#grant-trustline) to authorize the (customer) account rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn to hold issuances of USD from the (cold wallet) account rsA2LpzuawewSBQXkiju3YQTMzW13pAAdW:
 
-Request:
 
-```
-POST /v1/accounts/rsA2LpzuawewSBQXkiju3YQTMzW13pAAdW/trustlines?validated=true
-{
-  "secret": "sn3nxiW7v8KXzPzAqzyHXbSSKNuN9",
-  "trustline": {
-    "limit": "0",
-    "currency": "USD",
-    "counterparty": "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
-    "authorized": true
-  }
-}
-```
-_(**Reminder:** Don't send your secret to a server you do not control.)_
+The following is an example of using a locally-hosted `rippled`'s [`submit` command](rippled-apis.html#submit) to send a TrustSet transaction authorizing the (customer) account rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn to hold issuances of USD from the (cold wallet) account rsA2LpzuawewSBQXkiju3YQTMzW13pAAdW:
 
-Response:
+Request:
 
 ```
-201 Created
+POST http://localhost:8088/
 {
-  "success": true,
-  "trustline": {
-    "account": "rsA2LpzuawewSBQXkiju3YQTMzW13pAAdW",
-    "limit": "0",
-    "currency": "USD",
-    "counterparty": "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
-    "account_allows_rippling": true,
-    "account_trustline_frozen": false,
-    "authorized": true
-  },
-  "hash": "4509288EE17F01C83FC7D45850EB066A795EE5DBA17BB4DC98DD4023D31EEE5B",
-  "ledger": "11158585",
-  "state": "validated"
+    "method": "submit",
+    "params": [
+        {
+            "secret": "sn3nxiW7v8KXzPzAqzyHXbSSKNuN9",
+            "tx_json": {
+                "Account": "rsA2LpzuawewSBQXkiju3YQTMzW13pAAdW",
+                "Fee": "15000",
+                "TransactionType": "TrustSet",
+                "LimitAmount": {
+                    "currency": "USD",
+                    "issuer": "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
+                    "value": 0
+                },
+                "Flags": 65536
+            }
+        }
+    ]
 }
 ```
 
-A successful response shows `"state": "validated"` when the change has been accepted into a validated Ripple ledger.
```

Comment:
Known issue: This example is also missing a response, because I didn't have a successful test case. I can probably add one here later.